config.yaml

```diff
@@ -0,0 +1,124 @@
+# This file defines charm config options, and populates the Configure tab on Charmhub.
+# If your charm does not require configuration options, delete this file entirely.
+#
+# See https://juju.is/docs/config for guidance.
+
+options:
```

Comment:
Since the config values are explicitly for the users these will always be used to populate the AMS configuration regardless of whether we implement the corresponding relation to fallback or not. We can only probably only remove NRPE/nagios ones as we have dropped support for those.

Reply:
I mean we should start from scratch with no options and only add those we have implemented. As we're approaching feature parity we need to track which ones are missing and which we may want to drop.

Reply:
We still need to trim down the list to what is currently being used and leave all others out. We should have no option in here which is not yet implemented to be able to set the gap between the reactive and ops charm.

Reply:
@jat-canonical there are still options in here which we do no support at the moment, e.g. `nagios_context` or `nagios_servicegroups`, `storage_pool`, `storage_device`, `registry_mode`